priv/repo/migrations/20160907190212_create_location.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Gatherto.Repo.Migrations.CreateLocation do
+  use Ecto.Migration
+
+  def change do
+    create table(:locations) do
+      add :postal_code, :string
+      add :state, :string
+      add :city, :string
+      add :latitude, :float
+      add :longitude, :float
+
+      timestamps()
```

Comment:
Do we want a street address as well? I could see this being useful for both runs and clubs.